src/components/accordion/accordion.tsx

```diff
@@ -0,0 +1,30 @@
+import { Component, Element, h, Host, Listen } from "@stencil/core"
+
+@Component({
+  tag: "oui-accordion",
+})
+export class Accordion {
+  @Element() private self: HTMLElement
+
+  public render() {
+    return (
+      <Host>
+        <slot />
+      </Host>
+    )
+  }
+
+  /**
+   * Listen to `expand` event from child collapsable components
+   */
+  @Listen("expand")
```

Comment:
I feel like this could be structured so that all of the "toggling" shared logic could be handled in the same place rather than individual expanding in the collapsibles and collapsing happening here?

For instance maybe it would be good to lookup the sibling nodes from the collapsibles rather than lookup the children from the parent? Either that or put all the logic here..

Thoughts?